docs/rackhd/install_os.rst

```diff
@@ -140,7 +140,8 @@ Parameters  Type     Flags        Description
 ipAddr      String   **required** The assigned static IP address
 gateway     String   **required** The gateway.
 netmask     String   **required** The subnet mask.
-vlanIds     Array    *optional*   The VLAN ID. This is an array of integers (0-4095)
+vlanIds     Array    *optional*   The VLAN ID. This is an array of integers (0-4095).
+                                  In the case of Windows OS, the vlan is an array of one parameter only  
```

Comment:
Hi Felix,
As of now, the take the first value of the array and ignore the rest. 
I chose to keep the array in order to keep it consistent with the Linux implementation.